api-scanner/src/test/java/net/corda/plugins/AnnotatedMethodTest.java

```diff
@@ -44,9 +43,23 @@ public void testAnnotatedMethod() throws IOException {
 
         Path api = pathOf(testProjectDir, "build", "api", "annotated-method.txt");
         assertThat(api).isRegularFile();
-        assertThat(Files.readAllLines(api)).containsOnlyOnce(
-            "public class net.corda.example.HasAnnotatedMethod extends java.lang.Object",
-            "  @net.corda.example.A @net.corda.example.B @net.corda.example.C public void hasAnnotation()"
-        );
+        assertEquals(
```

Comment:
Again, I think that this test is more readable if we use AssertJ's `containsSequence()` now. (And oops, it looks like I missed the `UTF_8` for `Files.readAllLines()`!)